Makefile

```diff
@@ -274,14 +273,9 @@ stylesheets-check: stylesheets
 	fi;
 
 .PHONY: stylesheets
-stylesheets: public/css/index.css
-
-.IGNORE: public/css/index.css
-public/css/index.css: $(STYLESHEETS)
-	@which lessc > /dev/null; if [ $$? -ne 0 ]; then \
-		$(GO) get -u github.com/kib357/less-go/lessc; \
-	fi
-	lessc -i $< -o $@
+stylesheets:
+	node_modules/.bin/lessc --no-ie-compat public/less/index.less public/css/index.css
+	node_modules/.bin/cleancss -o public/css/index.css public/css/index.css
```

Comment:
We should output to `public/css/index.min.css` (need change also in template for using it) and keep `public/css/index.css` for readability.